sysroot_creator/scripts/create_cc_sysroot.py

```diff
@@ -219,56 +237,87 @@ def write_cc_system_setup_script(cc_root_dir: Path, platform: Platform):
     return cc_system_setup_script_path
 
 
+def setup_sysroot_environment(sys_setup_path: Path, build_setup_path:Path):
+    logger.info("Sourcing sysroot environment...")
+    logger.debug("Executing 'bash {}'".format(sys_setup_path))
+    subprocess.run(['bash', sys_setup_path])
+    logger.debug("Executing 'source {}'".format(build_setup_path))
+    subprocess.run(['source', build_setup_path])
+
+
 def create_arg_parser():
     parser = argparse.ArgumentParser()
     parser.add_argument(
-        '--arch',
+        '-a', '--arch',
         required=True,
         type=str,
+        choices=['armhf', 'aarch64'],
         help='Target architecture')
     parser.add_argument(
-        '--os',
+        '-o', '--os',
         required=True,
         type=str,
+        choices=['ubuntu', 'debian'],
         help='Target OS')
     parser.add_argument(
-        '--distro',
+        '-d', '--distro',
         required=False,
         type=str,
         default='dashing',
+        choices=['ardent', 'bouncy', 'crystal', 'dashing'],
         help='Target ROS distribution')
     parser.add_argument(
-        '--rmw',
+        '-r', '--rmw',
         required=False,
         type=str,
         default='fastrtps',
+        choices=['fatrtps', 'opensplice', 'connext'],
         help='Target RMW implementation')
     parser.add_argument(
         '--sysroot-base-image',
+        required=False,
         type=str,
-        help='Base image to use for building the sysroot')
+        help='Base Docker image to use for building the sysroot.'
+             'Ex. juanrh/ros2:armhf_bionic_crystal_fastrtps_prebuilt-crystal')
     parser.add_argument(
         '--docker-network-mode',
+        required=False,
         type=str,
         default='host',
         help="Docker's network_mode parameter to use for all "
              'Docker interactions')
     parser.add_argument(
         '--sysroot-nocache',
+        required=False,
         type=bool,
         default=False,
         help="When set to true, this disables Docker's cache when building "
              'the image for the workspace sysroot')
     parser.add_argument(
         '--force-sysroot-build',
+        required=False,
         type=bool,
         default=False,
         help='When set to true, we rebuild the sysroot and sysroot image, '
              'even if any of those is available')
+    parser.add_argument(
+        '--ros2-workspace',
+        required=False,
+        type=str,
+        default='./ros2_ws',
+        help='The location of the ROS2 workspace you\'ll be cross compiling against. '
+             'Usually ./ros2_ws if you moved it correctly.')
     return parser
 
 
+def update_pbar(pbar: tqdm, step: int) -> (tqdm, int):
```

Comment:
pbar -> progress_bar
Avoiding abbreviations let other users (or you in 6+ months) understand the code more easily.